File: app/src/main/java/ru/prsolution/winstrike/common/logging/MessageResponse.java
package ru.prsolution.winstrike.common.logging;
/*
 * Created by oleg on 07.03.2018.
 */
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class MessageResponse {

    @SerializedName("message")
    @Expose
    private String message;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

}

File: app/src/main/kotlin/ru/prsolution/winstrike/domain/models/MessageResponse.kt
package ru.prsolution.winstrike.domain.models

/*
 * Created by oleg on 07.03.2018.
 */
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class MessageResponse {

	@SerializedName("message")
	@Expose
	var message: String? = null

}

